### PR TITLE
Send new cli args to balsamic args for report generation

### PR DESCRIPTION
**Pull request by Mropat**
This PR adds/fixes 

cg workflow balsamic report-deliver now passes new options to balsamic:
--sample-id-map lims_id:tumor/normal:customer_sample_id
--case-id-map panel_shortname:case_name:application_tag

### How to prepare for test
- [x] ssh to relevant server (depending on type of change)
- [x] Use stage: `us`
- [x] paxa the environment: `paxa`
- [x] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [x] do cg workflow balsamic report-deliver unitedbeagle
- [x] do cg workflow balsamic report-deliver moralgoat

### Expected test outcome
- [x] check that balsamic runs well with new args
- [x] Take a screenshot and attach or copy/paste the output.

## Review
- [x] code approved by
- [x] tests executed by
- [x] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [x] **MINOR** - when you add functionality in a backwards compatible manner

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

**Review by moonso — APPROVED** [review]
🦩

File: cg/meta/workflow/balsamic.py [hunk 1]
```diff
@@ -373,6 +373,36 @@ def get_verified_config_case_arguments(
...
+        """Creates case info string for balsamic with format panel_shortname:case_name:application_tag}"""
+
+        case_obj: models.Family = self.store.family(case_id)
+        capture_kit = self.lims_api.capture_kit(case_obj.links[0].sample.internal_id)
+        if capture_kit:
+            panel_shortname = self.store.bed_version(capture_kit).shortname
+        elif self.get_application_type(case_obj.links[0]) == "wgs":
+            panel_shortname = "Whole_Genome"
+        else:
+            panel_shortname = "Undefined"
```
Comment (hassanfa) [inline cg/meta/workflow/balsamic.py:397]: Panel short name cannot be "Undefined" in the report. Please use the short name as it is defined here: https://clinical-api.scilifelab.se/admin/bedversion/

**Review by hassanfa — CHANGES_REQUESTED** [review]
Proper panel short name is required. Otherwise 👌🏽

**Comment by Mropat** [comment]
```
cg workflow balsamic report-deliver unitedbeagle
Creating delivery report for unitedbeagle
Found analysis finish file: /home/proj/stage/cancer/cases/unitedbeagle/analysis/analysis_finish
Running command /home/proj/bin/conda/envs/S_BALSAMIC/bin/balsamic report deliver --sample-config /home/proj/stage/cancer/cases/unitedbeagle/unitedbeagle.json --sample-id-map ACC6225A18:tumor:P3-D,ACC6225A14:normal:P3-germline --case-id-map GMCKsolid4.1:829728fam14:PANKTTR030

```

**Review by hassanfa — APPROVED** [review]
To the moooooon! 🚀 

Ta Maria!

**Comment by Mropat** [comment]
deployed in prod:
```
Building wheels for collected packages: cg
  Building wheel for cg (PEP 517) ... done
  Created wheel for cg: filename=cg-19.3.0-py3-none-any.whl size=397527 sha256=aa8136dcc3c7f9ab27ce22c89af2ffd22b7e366e13cd5ee1aa9ff2196e996844
  Stored in directory: /home/maryia.ropart/tmp/pip-ephem-wheel-cache-sa0udu1h/wheels/8c/1e/b8/816ed4facf8c5628b55966d3f2448eeb6ca70f35deff59389e
Successfully built cg
Installing collected packages: cg
  Attempting uninstall: cg
    Found existing installation: cg 19.2.0
    Uninstalling cg-19.2.0:
      Successfully uninstalled cg-19.2.0
Successfully installed cg-19.3.0

```

**Pull request merged**